src/UserAgreement/Option.jsx

```diff
@@ -0,0 +1,50 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import './Option.less';
+
+function getCharFromA(i) {
+  return String.fromCharCode('A'.charCodeAt(0) + i);
+}
+
+class Option extends Component {
+  static propTypes = {
+    option: PropTypes.string.isRequired,
+    idx: PropTypes.number.isRequired,
+    isCorrectAnswer: PropTypes.bool.isRequired,
+    onChange: PropTypes.func.isRequired,
+    selected: PropTypes.bool.isRequired,
+  };
+
+  constructor(props) {
+    super(props);
+    this.answer = null;
+  }
+
+  render() {
+    let frameClassModifier = '';
+    let bulletClassModifier = '';
+    if (this.props.selected) {
+      frameClassModifier = this.props.isCorrectAnswer ? 'option__frame--correct' : 'option__frame--incorrect';
+      bulletClassModifier = this.props.isCorrectAnswer ? 'option__bullet--correct' : 'option__bullet--incorrect';
+    }
+    return (
+      <div
+        className={`option__frame ${frameClassModifier}`}
+        role='button'
+        onClick={() => this.props.onChange(this.props.idx)}
+        onKeyDown={this.handleKeyDown}
+        tabIndex={-10}
```

Comment:
Is this just a placeholder value? is it possible to use a real `tabIndex` to make it accessible? :)

Reply:
codacy suggests a negative value for the button